examples/typescript/bulkDisableSchedules.ts

```diff
@@ -7,28 +7,27 @@ const sdk = LookerNodeSDK.init40()
 
 const getSchedules = async (allUsers: boolean, userID?: number): Promise<IScheduledPlan[]> => {
```

Comment:
Shouldn't `allUsers` be removed as a parameter?

Reply:
Yeah, since solely userID is being used as the check will remove `allUsers` as a parameter in getSchedules()